### DISCUSSION TITLE
How about preparing a production ready docker env?

### DISCUSSION THREAD

**Discussion by mucan54** [markshust/docker-magento#1021] [Ideas]
I believe there is a lack of sufficient resources on preparing a production-ready Docker Compose configuration for Magento. The existing Magento Docker environment for local development has undergone extensive testing, updates, and maintenance over time. Hence, I propose initiating a new project that leverages this wealth of knowledge and community support to deliver an open-source, production-ready Docker configuration for Magento.

**Comment by PrashantPunter** [comment]
I was thinking about the same as there is no reliable production-ready Docker Compose configuration for Magento.
